app/src/main/java/sdp/moneyrun/map/MapActivity.java

```diff
@@ -244,6 +256,22 @@ public void removeCoin(Coin coin) {
         }
     }
 
+    public void moveCameraTo(float latitude, float longitude){
```

Comment:
This function is already in the TrackedMap class, I moved it there because there were too many functions in MapActivity, so you should remove either one of them, I would suggest this one so there are fewer functions in MapActivity but up to you.